manifests/03-rbac-role-ns-operator.yaml

```diff
@@ -9,6 +9,7 @@ rules:
   resources:
   - configmaps
   - events
+  - secrets
```

Comment:
Are we creating a new secret?

Reply:
Ah, I see, `v1helpers.CachedSecretGetter(kubeClient.CoreV1(), resourceSyncerInformers),`
We have to give the syncer a secretGetter, even though we aren't copying a secret.